pkg/server/wireexts_oss.go

```diff
@@ -88,6 +89,8 @@ var wireExtsBasicSet = wire.NewSet(
 	wire.Bind(new(accesscontrol.DatasourcePermissionsService), new(*ossaccesscontrol.DatasourcePermissionsService)),
 	secretsStore.ProvideRemotePluginCheck,
 	wire.Bind(new(secretsStore.UseRemoteSecretsPluginCheck), new(*secretsStore.OSSRemoteSecretsPluginCheck)),
+	secretsMigrations.ProvideSecretMigrationServiceProvider,
```

Comment:
Could this just go in wire.go instead of the oss extensions?

Reply:
Good point, will try it out, I followed the same pattern the backwards services have that's why it ended up in this file.

Reply:
Done on https://github.com/grafana/grafana/pull/50463/commits/3f40ccccd7724bb4aaa93b7821b77142ef2bb35e